pkg/resourceinterpreter/utils/utils.go

```diff
@@ -0,0 +1,22 @@
+package utils
+
+import (
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+
+	"github.com/karmada-io/karmada/pkg/util"
+)
+
+// ApplyReplica applies the Replica value for the specific field.
+func ApplyReplica(workload *unstructured.Unstructured, desireReplica int64, field string) error {
```

Comment:
This function just takes `desireReplica` as the common value, it cloud be used to set any fields with `int64` type, right?
I mean the common function is misleading.

By the way, I don't think we should add a `utils` package here, too many `utils` in the same repo is also confusing, we have to use ugly import alias to distinguish them.
Not sure if we need this function yet, if it does, it should be `https://github.com/karmada-io/karmada/blob/master/pkg/util/helper/unstructured.go`.

Reply:
> it cloud be used to set any fields with int64 type, right?

Yes any fields under `spec`, so should we rename it? Like `ApplySpecInt64Value`, it's weird and there are no others who will use this function.

> it should be https://github.com/karmada-io/karmada/blob/master/pkg/util/helper/unstructured.go.

Sure.